src/components/CreateGroupDialog.vue

```diff
@@ -33,15 +33,27 @@
 import { Modal } from "bootstrap";
 
 export default {
+    beforeRouteLeave(to, from, next) {
```

Comment:
are there cases where the `beforeUnmount` is not called?
If yes, please add a comment what this case is.
If no, lets remove this check and inline `cleanupModal` into `beforeUnmount`

Reply:
If the user refreshes the page or navigates to another URL using the browser's address bar, the Vue instance might not clean up properly, meaning beforeUnmount won't be triggered. correct me if i am wrong

Reply:
btw all of cleanupModal shud be made inline right if i am not wrong

Reply:
hey so only one hook is calling the function cleanupModal, shud i make it inline? @CommanderStorm